pnpm-lock.yaml

```diff
@@ -0,0 +1,3326 @@
+lockfileVersion: 5.4
```

Comment:
please don't add a pnpm lockfile. we'll probably switch to pnpm before too long, but you should use npm for now

Reply:
My apologies! First time contributor, didn't mean to commit it.